crates/teloxide-core/schema.ron

```diff
@@ -2582,7 +2587,7 @@ Schema(
         ),
         Method(
             names: ("editChatInviteLink", "EditChatInviteLink", "edit_chat_invite_link"),
-            return_ty: String,
+            return_ty: RawTy("ChatInviteLink"),
```

Comment:
I think this fixed bug should be mentioned in the changelog and maybe migration guide as breaking change